website/docs/MocksAndSpies.md

```diff
@@ -7,7 +7,9 @@ WebdriverIO comes with built in support for modifying network responses that all
 
 :::info
 
-This feature is currently only supported when running local tests on Chrome. It is planned to be supported on [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1591389) and [Sauce Labs](https://saucelabs.com/) soon. If you encounter problems using it please file [an issue](https://github.com/webdriverio/webdriverio/issues/new/choose) and let us know!
+Note that using the `mock` command requires support for Chrome DevTools protocol.
```

Comment:
We should add a note that cross browser support is expected once required primitives landed in WebDriver Bidi and got implemented in respective browser.